### ISSUE TITLE
Allow HUD customization without overwriting files.

### ISSUE THREAD

**Issue by squeek502** [fortressforever/fortressforever#297]
Related to https://github.com/fortressforever/fortressforever/issues/95

Something I noticed while working on #296 is that some HUD-related script files are loaded in the code by path, so it might be possible to conditionally load a separate path if it exists. This would allow us, for example, to load `FortressForever/custom/scripts/HudLayout.res` if it exists, and `FortressForever/scripts/HudLayout.res` if it doesn't, which would in turn allow users to have custom HUD scripts that wouldn't get overwritten by game updates/verifying game files/changing Steam branches/etc.

There's at least a few files we'd need to do this for. Off the top of my head:
- HudLayout.res
- Scheme file(s) (SourceScheme.res/ClientScheme.res)
- HudAnimations.txt

**Comment by doublah** [comment]
Firstly, in source 2007 games, custom files are loaded in a method like such: `FortressForever/custom/customhud/scripts/HudLayout.res`. The `customhud` folder here can be named anything by the end user and still works.

Secondly, this MIGHT make it theoretically possible to be able to change HUDs from in-game, without restarting or exiting. This could be another useful feature.

**Comment by squeek502** [comment]
Yeah, I considered that as well. Could have a cvar controlling which subfolder to use.
